test/libsolidity/syntaxTests/pragma/experimental_solidity_out_of_order_2.sol

```diff
@@ -0,0 +1,13 @@
+function f() pure returns (uint)
```

Comment:
Not much, except that this one doesn't have `pragma experimental solidity` at the end, but in the middle. It could be removed, but also doesn't do much harm. @ekpyron?

Reply:
Fine either way - one more test doesn't hurt, syntax tests are cheap, so just keep it.